<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>import_settings</class>
 <widget class="QMainWindow" name="import_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Arc NeuroWave Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string notr="true">Select a sample rate...</string>
    </property>
    <property name="maxVisibleItems">
     <number>9</number>
    </property>
    <property name="placeholderText">
     <string>12321</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Select a data type...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Type</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample rate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
